### DISCUSSION TITLE
Getting Invalid Signature

### DISCUSSION THREAD

**Discussion by suyogyabluebeam** [ory/hydra#2282] [Q&A] [answered]
Did a deployment in EKS (using the helm chart ory/hydra). When I try to verify access token "jwt" in jwt.io, I get invalid signature.
- Currently using generated keys for signing.

**Comment by vinckr** [comment, marked as answer]
You created the jwt with Ory Hydra? I have not used the Helm charts before, but I had no problems with a "normal" install.

**Reply by suyogyabluebeam** [reply]
Yes. Turns out this was a non issue. I wasn't using the right public key.

**Reply by vinckr** [reply]
Nice, glad it got resolved so easily 😄 . 
Can you choose your answer as solving this @suyogyabluebeam ?
